app/views/RoomView/index.tsx

```diff
@@ -1018,8 +1018,12 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 				await Promise.race([this.list.current?.jumpToMessage(message.id), new Promise(res => setTimeout(res, 5000))]);
 				this.cancelJumpToMessage();
 			}
-		} catch (e) {
-			log(e);
+		} catch (error: any) {
+			if (isFromReply && error.data?.errorType === 'error-not-allowed') {
+				showErrorAlert(I18n.t('The_room_does_not_exist'), I18n.t('Room_not_found'));
+			} else {
+				log(error);
+			}
```

Comment:
I don't have enough information to determine if every time the error is 'error-not-allowed' in this case, 'The_room_does_not_exist' should appear.

That's why I decided to add the 'isFromReply' condition.